tests/nanoemoji_test.py

```diff
@@ -210,69 +209,3 @@ def test_omit_empty_color_glyphs():
         "omit_empty_color_glyphs.ttx",
         include_tables=["GlyphOrder", "cmap", "glyf", "COLR", "SVG "],
     )
-
```

Comment:
why did you delete this whole chunk of test file below? There are two test methods which I think are unrelated to your PR

Reply:
I amended your last commit to restore the deleted tests